experimental/Schemes/IdealSheaves.jl

```diff
@@ -487,3 +487,59 @@ function order_on_divisor(
 #    order_dict[U] = upper-lower
 end
 
+@attr IdealSheaf function radical(II::IdealSheaf)
+  X = scheme(II)
+  # If there is a simplified covering, do the calculations there.
+  covering = (has_attribute(X, :simplified_covering) ? simplified_covering(X) : default_covering(X))
+  ID = IdDict{AbsSpec, Ideal}()
+  for U in patches(covering)
+    ID[U] = radical(II(U))
+  end
+  return IdealSheaf(X, ID, check=false)
+end
+
+@attr MPolyLocalizedIdeal function radical(I::MPolyLocalizedIdeal)
+  J = pre_saturated_ideal(I)
+  return ideal(base_ring(I), gens(radical(J)))
+end
+
+@attr MPolyQuoLocalizedIdeal function radical(I::MPolyQuoLocalizedIdeal)
+  W = base_ring(I)
+  J = pre_image_ideal(I)
+  return ideal(W, [g for g in W.(gens(radical(J))) if !iszero(g)])
+end
+
+@Markdown.doc """
+    smooth_lci_covering(I::IdealSheaf)
```

Comment:
No abbreviations please.

Reply:
@simonbrandhorst: Suggest a better name for a function finding a covering such that the (smooth) locally complete intersection is described by a complete intersection on each patch. This is a vital helper for many practical computations.

Reply:
`complete_intersection_cover(I::IdealSheaf)`?

Reply:
Maybe some consistency is in order here as this pattern shows up in several places. 
I suggest the prefix `as_locally_bla` for such things e.g.: `as_locally_complete_intersection`, `as_locally_free_module` 
`as_locally_principal_ideal`. The drawback is that it hides that the functions actually return a cover. But at least it is consistent.  In any case I would strongly argue for a  consistent naming. (whatever it turns out to be)